ui/k-toolbar/main.kytos

```diff
@@ -202,19 +405,53 @@ module.exports = {
         if (this.queue_id !== undefined && this.queue_id !== "") {
             request.queue_id = parseInt(this.queue_id)
         }
+
+        ['primary_constraints', 'secondary_constraints'].forEach(_type => {
+          request[_type] = {};
+          request[_type].flexible_metrics = {};
+          request[_type].mandatory_metrics = {};
+
+          // mandatory and flexible metrics
+          ['bandwidth', 'utilization', 'priority', 'delay', 'ownership'].forEach(_m => {
+            if (_this.is_flexible[_type][_m]) {
+              request[_type].flexible_metrics[_m] = _this.metrics[_type][_m];
+            } else {
+              request[_type].mandatory_metrics[_m] = _this.metrics[_type][_m];
+            }
+          });
+
+          // Dropdown forms
+          if(this.form_constraints[_type].desired_links.length > 0) {
+            request[_type].desired_links = this.form_constraints[_type].desired_links;
```

Comment:
@rmotitsuki as we discussed earlier, since there seems to have a bug (https://github.com/kytos-ng/ui/issues/34) on the UI's `k-select` component (which per our conversation we decided not include in this release due to the colateral effect), the following suggestions aims to workaround such bug and only submit the actual selected elements:
```suggestion
            request[_type].desired_links = this.form_constraints[_type].desired_links.filter(item => typeof(item) == "string");
```